Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor

```diff
@@ -1,6 +1,419 @@
+
+        <OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBar>
+            <OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBarLeft>
+                <Button Class="menu-icon" data-open="offCanvasLeft"></Button>
+                Balzor Foundation
+            </OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBarLeft>
+            <OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBarRight>
+                <Button Class="menu-icon" data-open="offCanvasRight"></Button>
+            </OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBarRight>
+        </OpenCodeDev.Blazor.Foundation.Components.Plugins.Sticky.StickyTitleBar>
+<OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas Position="position-left" Id="offCanvasLeft" style="background-color: #111;">
+    <ul style="color:#ffffff">
+        <li>About</li>
+        <li>Services</li>
+        <li>Clients</li>
+        <li>Contact</li>
+    </ul>
+</OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas>
+<OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas Position="position-left" Id="offCanvasRight">
+    <ul>
+        <li>About</li>
+        <li>Services</li>
+        <li>Clients</li>
+        <li>Contact</li>
+    </ul>
+</OpenCodeDev.Blazor.Foundation.Components.Containers.OffCanvas>
+
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="primary" style="margin-top: 50px;">
+            When using the title-bar with a vertical off-canvas, the StickyTitleBar icons are still either StickyTitleBarLeft or StickyTitleBarRight.
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Nested Off-Canvas</h2>
+        <p>
+            Another improvement is the support of several off-canvas elements that share the same position e.g. two elements with <Code>position-left</Code>
+        </p>
+        <p>
+            Advanced off-canvas users may use the new contentId option to bind an element to a content. This lets you place the element much more flexibly as it may be a sibling of the content, a child or none of it.<br />
+            <b>Important:</b> when using the contentId on a nested element you must also use the new nested option and tell the JavaScript it's nested!
```

Comment:
remove reference to JS no need to talk about JS

Reply:
```suggestion
           <p>
            Advanced off-canvas users may use the new contentId option to bind an element to a content. This lets you place the element much more flexibly as it may be a sibling of the content, a child or none of it.<br />
        </p>
```